### PR TITLE
Hide DtBuilder implementation

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: src/backend/dt.c [hunk 1]
```diff
@@ -27,6 +27,13 @@ static char __file__[] = __FILE__;      /* for tassert.h                */
 
 static dt_t *dt_freelist;
 
+static dt_t **dtabytes(dt_t **pdtend,tym_t ty, unsigned offset, unsigned size, const char *ptr, unsigned nzeros);
```
Comment (yebblies) [inline src/backend/dt.c]: Can these be inlined into the methods in `DtBuilder`?

Comment (WalterBright) [inline src/backend/dt.c]: That's what I'm doing.

**Comment by WalterBright** [comment]
ready to merge

**Comment by yebblies** [comment]
Auto-merge toggled on

**Pull request merged**